packages/frontend/.env.example

```diff
@@ -1,3 +1,3 @@
 PORT=3000
-NEXT_PUBLIC_API_URL="http://localhost:5000/public/v1"
-NEXT_PUBLIC_CAPTCHA_SITE_KEY=your-google-recaptcha-sitekey
+API_HOSTNAME="localhost"
+API_PORT=5000
```

Comment:
Sorry, just wondering, does this variable work in browser side? Cause I see you are also giving default value in implementation. From what I know, next env variable has to start with "NEXT_PUBLIC_" in the name so that it can be used in browser side

Reply:
I'll check.